docs/serving/encryption/configure-certmanager-integration.md

```diff
@@ -0,0 +1,246 @@
+        name: selfsigned-cluster-issuer
+    ```
+
+1. Ensure that the `ClusterIssuer` is ready:
+
+    ```bash
+    kubectl get clusterissuer selfsigned-cluster-issuer -o yaml
+    kubectl get clusterissuer knative-selfsigned-issuer -o yaml
+    ```
+    Result: The `Status.Conditions` should include `Ready=True`.
+
+1. Then reference the `ClusterIssuer` in the `config-certmanager` ConfigMap:
+
+    ```bash
+    kubectl edit configmap config-certmanager -n knative-serving
+    ```
+
+    Add the fields within the `data` section:
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: config-certmanager
+      namespace: knative-serving
+      labels:
+        networking.knative.dev/certificate-provider: cert-manager
+    data:
+      issuerRef: |
+        kind: ClusterIssuer
+        name: knative-selfsigned-issuer
+      clusterLocalIssuerRef: |
+        kind: ClusterIssuer
+        name: knative-selfsigned-issuer
+      systemInternalIssuerRef: |
+        kind: ClusterIssuer
+        name: knative-selfsigned-issuer
+    ```
+
+    Ensure that the file was updated successfully:
+
+    ```bash
+    kubectl get configmap config-certmanager -n knative-serving -o yaml
+    ```
+
+
+## Managing trust and rotation without downtime
+
+As pointed out above, each client that calls a Knative Service using HTTPS needs to trust the CA and/or intermediate chain.
+If you take a look at the [encryption overview](./encryption-overview.md), you can see that there are multiple places where 
+trust needs to be distributed:
+
+* **Cluster external client (Browser and/or other application)**: this is considered out of scope of Knative.
+* **Cluster internal client (e.g. Knative or Vanilla K8s workload)**: see below.
+* **Knative system components (e.g. Activator, Queue-Proxy, Ingress-Controller)**: see below.
+
+
+### Trusting for cluster internal clients (e.g. Knative or Vanilla K8s workload)
+
+As Knative does not control all workload and the settings are highly dependent on your runtime and/or language, this is out of scope of Knative.
+But here a few points to consider, as there are several ways on how to provide CAs to your application:
+
+* Adding the CA bundle to a Container image on build-time (be aware that this complicates CA rotation, you basically need to rebuild every application)
+* Mounting a CA bundle to the filesystem (e.g. from a `Secret` or `ConfigMap`)
+* Reading it from environment variable
+* Accessing it from a `Secret`/`ConfigMap` via K8s API
+
+If reloading certificates without downtime is important for your client, the workload must either watch changes on the K8s resource (Secret/ConfigMap) or watch the filesystem.
+If the workload is watching the filesystem, it is important to note that using `ionotify` to catch changing Secrets/ConfigMaps is not very reliable on K8s. Tests have shown that it is more reliable to regularly poll and check the certificate on the filesystem for changes.
```

Comment:
We will need to have an example on how to do the rotation. I think a minimum example should include manual rotation based on workload restarting (as others have).